packages/public/inspection-capture-web/src/PhotoCapture/PhotoCaptureHUDAddDamagePreview/CrosshairPreview/CrosshairPreview.tsx

```diff
@@ -0,0 +1,40 @@
+import { Button, DynamicSVG } from '@monkvision/common-ui-web';
+import { useTranslation } from 'react-i18next';
+import { useState } from 'react';
+import { AddDamagePreviewMode, usePhotoHUDButtonBackground } from '../../hooks';
+import { styles } from './CrosshairPreview.styles';
+import { DamageCounter } from '../DamageCounter';
+import { CancelButton } from '../CancelButton';
+
+export interface CrosshairPreviewProps {
+  onCancel: () => void;
+}
+
+export function CrosshairPreview({ onCancel }: CrosshairPreviewProps) {
+  const [showInfoBtn, setShowInfoBtn] = useState(true);
+
+  const { t } = useTranslation();
+  const { bgColor } = usePhotoHUDButtonBackground();
+  const addDamagePreviewMode = AddDamagePreviewMode.DEFAULT;
```

Comment:
What is the point of this variable? You can remove it I think

Reply:
It's for this component: `<DamageCounter addDamagePreviewMode={addDamagePreviewMode} />`
I wanted to make it more lisible, but actually it's useless
Instead,  I can pass it directly as a props: `addDamagePreviewMode={AddDamagePreviewMode.DEFAULT}`